db/migrate/20160829175601_add_content_filter_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddContentFilterToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :content_filter, :string
```

Comment:
I think I agree with Joe on this point.  This is a perfect example of a single complex piece of data attached to the user which you will never need to search or filter on.  The only time anyone ever looks at this is when you need to filter content or adjust the filter parameters.  I see no advantage in making a particular combination of filter settings a separate stand-alone entity which can be shared by multiple users.  Instead I think it is more natural to think of this as a description of a filter.  Each user can describe their own unique filter.  It always seemed to me that a serialized hash would be the natural implementation.  Especially since a hash like that can be passed in verbatim to any observation query.  Then it's just a matter of creating a generalized and extensible UI for adjusting this hash of filter parameters appropriately.